<!-- -->
<launch>
	<node pkg="data_collect" type="imu_saver" name="imu_saver_node" output="screen" clear_params="true">
		<param name="imu_topic" type="string" value="/imu/data" />
		<param name="output_path" type="string" value="$(find data_collect)/output/imu/" />
	</node>
</launch>

